addOns/imagelocationscanner/imagelocationscanner.gradle.kts

```diff
@@ -1,6 +1,6 @@
 import org.zaproxy.gradle.addon.AddOnStatus
 
-version = "2"
```

Comment:
Version 2 was not yet released.

Reply:
ok.  i thought i was supposed to update this.  i'll undo.